resources/com/mesosphere/global/terraform_file_deploy.sh

```diff
@@ -152,8 +172,8 @@ EOF
 }
 EOF
   echo -e "\e[32m deployed nginx \e[0m"
-  timeout -t 120 bash <<EOF || ( echo -e "\e[31m failed to reach nginx... \e[0m" && exit 1 )
-while ${TMP_DCOS_TERRAFORM}/dcos marathon app show nginx | jq -e '.tasksHealthy != 1' > /dev/null 2>&1; do
```

Comment:
please do not change

Reply:
fixed at 1bd4b9127b8d219de1e3ce02dd03ee08e36041f3